pulsar/consumer_partition.go

```diff
@@ -21,6 +21,7 @@ import (
 	"fmt"
 	"math"
 	"sync"
+	syncAtomic "sync/atomic"
```

Comment:
perhaps you don't need to set alias to atomic package, just use atomic

Reply:
@jonyhy96 ,

Good suggestion; I've cut over to using [uber-go.atomic.Value](https://github.com/uber-go/atomic/blob/master/value.go)